dongkyu/initial-settings-express/myapp/app.js

```diff
@@ -31,12 +30,65 @@ app.use(cors());
 app.use(morgan("dev"));
 
 
-//health check
 app.get("/ping", (req, res) => {
   return res.status(200).json({ message: "pong" });
 });
 
 
+app.post("/users", async (req, res) => {
+  const { name, age, email, password } = req.body
+
+  await appDataSource.query(
+    `INSERT INTO users(
+      name,
+      age,
+      email,
+      password
+    ) VALUES (?, ?, ?, ?);  
+    `,
+    [name, age, email, password]
+  );
+
+  res.status(201).json({ message : "userCreated" });
+})
+
+
+app.post("/posts", async (req, res) => {
+  const { title, content, user_id } = req.body
+
+  await appDataSource.query(
+    `INSERT INTO posts(
+      title,
+      content,
+      user_id
+    ) VALUES (?, ?, ?);
+    `,
+    [title, content, user_id]
+  );
+
+  res.status(201).json({ message : "postCreated"})
+})
+
+
+app.get("/users_posts", async (req, res) => {
+  await appDataSource.manager.query(
```

Comment:
```suggestion
  await appDataSource.manager.query(
```
해당 query method만 `app.DataSource.manager.query()`로 작성하신 이유가 있을까요?